frontends/infinite-corridor/src/pages/FieldPage.tsx

```diff
@@ -66,17 +65,14 @@ const FieldCarousel: React.FC<FieldListProps> = ({ list }) => {
       cellSpacing={0} // we'll handle it with css
       title={<h3>{list.title}</h3>}
       previous={
-        <IconButton
-          title="Show previous courses"
-          className="ic-carousel-button"
-        >
-          <NavigateBeforeIcon />
-        </IconButton>
+        <button type="button" className="ic-carousel-button outlined-button">
+          <ArrowBack fontSize="inherit" /> Previous
+        </button>
```

Comment:
This and the related changes make the carousel buttons similar to learn
<img width="220" alt="Screen Shot 2022-08-15 at 10 31 21 AM" src="https://user-images.githubusercontent.com/9010790/184655010-29d05fc2-2bd5-4449-92d6-d419b1e7eb04.png">
instead of:
<img width="105" alt="Screen Shot 2022-08-15 at 10 31 52 AM" src="https://user-images.githubusercontent.com/9010790/184655107-af2d1826-badc-4d30-a0b8-4f7c5af019cd.png">